locales/en-US/translation.yml

```diff
@@ -332,16 +353,60 @@ admin:
     build_description3: This may take a while.
   share:
     legend: External Share Settings
-    enable_external_share: Wikiページの外部ユーザーへの共有を有効にする。
+    enable_external_share: Enable sharing Wiki pages to external users.
   user:
     legend: User List
-    user_id: ユーザーID
-    name: 名前
-    email: メールアドレス
-    created_at: 作成日
-    status: ステータス
-    operation: 操作
-    not_found: ユーザーが見つかりませんでした
+    user_id: User ID
+    name: Name
+    email: Email
+    created_at: Created
+    status: Status
+    operation: Operation
+    not_found: User Not Found
+    invite:
+      form:
+        collapse: Invite new users
+        email: Email
+        example: 'ex. user@crowi.wiki'
+        description: You can invite more than one person at once with multi-line email addresses.
+        checkbox: Send invitation by email
+        submit: Submit
+      modal:
+        completed_message: Success for inviting new users
+        temp_password: The new users have temporary passwords.
+        caution_password: Please note that you can never find temporary passwords after you close this message.
```

Comment:
Please note you will not be able to recover the temporary passwords once you close this message.